util/dvsim/dvsim.py

```diff
@@ -383,10 +383,10 @@ def parse_args():
 
     rung.add_argument("--verbosity",
                       "-v",
-                      choices=['n', 'l', 'm', 'h', 'd'],
+                      choices=['n', 'l', 'm', 'h', 'f', 'd'],
                       metavar='V',
                       help=('Set tool/simulation verbosity to none (n), low '
-                            '(l), medium (m), high (h) or debug (d). '
+                            '(l), medium (m), high (h) full (f) or debug (d). '
```

Comment:
Nit: If re-pushing, this needs a comma before "full"

Reply:
Done.